backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/calendar/standard/CalendarId.java

```diff
@@ -39,19 +41,20 @@ public static CalendarId ofRepoId(final int repoId)
 		return new CalendarId(repoId);
 	}
 
+	@Nullable
 	public static CalendarId ofRepoIdOrNull(final int repoId)
 	{
 		return repoId > 0 ? new CalendarId(repoId) : null;
 	}
 
-	public static int toRepoId(final CalendarId CalendarId)
+	public static int toRepoId(@Nullable final CalendarId calendarId)
 	{
-		return CalendarId != null ? CalendarId.getRepoId() : -1;
+		return calendarId != null ? calendarId.getRepoId() : -1;
 	}
 
 	private CalendarId(final int repoId)
 	{
-		this.repoId = Check.assumeGreaterThanZero(repoId, "repoId");
+		this.repoId = Check.assumeGreaterThanZero(repoId, "calendarId");
```

Comment:
leave it like this but usually here i prefer to use the column name as message,
i.e. `"C_Calendar_ID"` instead of `"calendarId"`
because it's much more descriptive for whom will troubleshoot/debug.

Reply:
done.